.github/workflows/lint-code.yml

```diff
@@ -2,9 +2,9 @@ name: Lint code
 
 on:
   push:
-    branches: [master, next, next-major, beta, alpha]
+    branches: [master, next, next-major, beta, alpha, '[0-9]+.x']
   pull_request:
-    branches: [master, next, next-major, beta, alpha]
+    branches: [master, next, next-major, beta, alpha, '[0-9]+.x']
```

Comment:
Maybe we can delete this line so it applies to all PRs in general

Reply:
good point! the `push` is also redundant since the only way any of those branches get commits is via approved merged PRs. updated.